sympy/physics/units/dimensions.py

```diff
@@ -414,126 +417,107 @@ def get_dim(self, dim):
                     break
         elif isinstance(dim, Dimension):
             for i, idim in enumerate(self._dims):
-                if dim.get_dimensional_dependencies() == idim.get_dimensional_dependencies():
+                if dim.get_dimensional_dependencies() == \
+                        idim.get_dimensional_dependencies():
                     return idim
 
         return found_dim
 
     def extend(self, base, dims=(), name='', description=''):
-        """
-        Extend the current system into a new one.
+        """Extend the current system into a new one.
 
         Take the base and normal units of the current system to merge
-        them to the base and normal units given in argument.
-        If not provided, name and description are overriden by empty strings.
+        them to the base and normal units given in argument. If not
+        provided, name and description are overriden by empty strings.
         """
-
         base = self._base_dims + tuple(base)
         dims = self._dims + tuple(dims)
 
         return DimensionSystem(base, dims, name, description)
 
     @staticmethod
     def sort_dims(dims):
-        """
-        Sort dimensions given in argument using their str function.
+        """Sort dimensions given in argument using their str function.
 
-        This function will ensure that we get always the same tuple for a given
-        set of dimensions.
+        This function will ensure that we get always the same tuple for
+        a given set of dimensions.
         """
-
         return tuple(sorted(dims, key=str))
 
     @property
     def list_can_dims(self):
-        """
-        List all canonical dimension names.
-        """
+        """List all canonical dimension names."""
         dimset = set([])
         for i in self._base_dims:
             dimset.update(set(i.get_dimensional_dependencies().keys()))
         return tuple(sorted(dimset))
 
     @property
     def inv_can_transf_matrix(self):
-        """
-        Compute the inverse transformation matrix from the base to the
-        canonical dimension basis.
+        """Compute the inverse transformation matrix.
 
-        It corresponds to the matrix where columns are the vector of base
-        dimensions in canonical basis.
+        It is used for transformation from the base to the canonical
+        dimension basis. It corresponds to the matrix where columns are the
+        vector of base dimensions in canonical basis.
 
         This matrix will almost never be used because dimensions are always
         define with respect to the canonical basis, so no work has to be done
         to get them in this basis. Nonetheless if this matrix is not square
         (or not invertible) it means that we have chosen a bad basis.
         """
-
         matrix = reduce(lambda x, y: x.row_join(y),
                         [self.dim_can_vector(d) for d in self._base_dims])
 
         return matrix
 
-    #@cacheit
+    # @cacheit
```

Comment:
Just remove that. Or turn into a useful note: `NOTE : This may be able to be cached.`.